spmp.sh

```diff
@@ -0,0 +1 @@
+LD_LIBRARY_PATH="/app/lib" /app/dist/bin/spmp --flatpak "$@"
```

Comment:
you don't need to set library path.

Reply:
It doesn't work without that (tested it again now just to be sure), so I'd say I do need to set it.

Reply:
> It doesn't work without that (tested it again now just to be sure), so I'd say I do need to set it.

What's the issue? Flatpak handles this by default.

Reply:
LD_LIBRARY_PATH is used by Java to set `java.library.path`, which is used to find and load libraries dynamically at runtime. Whatever Flatpak does doesn't cover this case I guess,